lifecycle-operator/controllers/common/evaluation/evaluationhandler_test.go

```diff
@@ -265,14 +266,7 @@ func TestEvaluationHandler(t *testing.T) {
 				},
 			}
 			fakeRecorder := record.NewFakeRecorder(100)
-			handler := EvaluationHandler{
-				SpanHandler: &spanHandlerMock,
-				Log:         ctrl.Log.WithName("controller"),
-				EventSender: NewK8sSender(fakeRecorder),
-				Client:      fake.NewClientBuilder().WithObjects(&tt.evalObj).Build(),
-				Tracer:      trace.NewNoopTracerProvider().Tracer("tracer"),
-				Scheme:      scheme.Scheme,
-			}
+			handler := NewEvaluationHandler(fake.NewClientBuilder().WithObjects(&tt.evalObj).Build(), common.NewK8sSender(fakeRecorder), ctrl.Log.WithName("controller"), trace.NewNoopTracerProvider().Tracer("tracer"), scheme.Scheme, &spanHandlerMock)
```

Comment:
please split this up so that this is not all in one line, this will make it a bit easier to read

Reply:
Alright, let me do that now.